project/dexter-metrics/src/test/DexterMetricsActivatorTest.java

```diff
@@ -0,0 +1,47 @@
+
+/**
+ * Copyright (c) 2014 Samsung Electronics, Inc.,
+ * All rights reserved.
+ * 
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * 
+ * * Redistributions of source code must retain the above copyright notice, this
+ *   list of conditions and the following disclaimer.
+ * 
+ * * Redistributions in binary form must reproduce the above copyright notice,
+ *   this list of conditions and the following disclaimer in the documentation
+ *   and/or other materials provided with the distribution.
+ * 
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+import static org.junit.Assert.assertNull;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.osgi.framework.*;
+
+public class DexterMetricsActivatorTest {
+
+	@Test
+	public void stopTest_ReturnsNullAndThrowsNoExceptions() {
+		DexterMetricsActivator tester = new DexterMetricsActivator();
+		BundleContext bundleContext = null;
+		try {
+			tester.stop(bundleContext);
+			assertNull(DexterMetricsActivator.getContext());
+		} catch (Exception e) {
```

Comment:
Exception checking is not needed  because the function 'stop()' doesn't throw any Exception.
And it would be better to delete the statement 'throws Exception'  in function definition of 'stop()' (DexterMetricsActivator.java 49 line).